### PR TITLE
New Adapter: TradPlus

### PR DESCRIPTION
**Pull request by tradplus**
TradPlus is a leading, stable, and reliable ad monetization platform that is committed to providing global developers with fair, transparent and efficient monetization solutions. We also provide ADX and Saas ADX services. We serve 2,000+ global developers. Moreover, Our SDK has been certified by IAB Tech Lab and also joins Google Play SDK Index. And we have Information Security Management Certification.

Our advertising business spans the world. This is our official website: https://tradplusad.com/en. You can get more details about TradPlus.

this is TradPlus adapter version 1.0.

Thank!

### CODE REVIEW

File: src/test/java/org/prebid/server/bidder/tradplus/TradPlusBidderTest.java [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+        final BidRequest bidRequest = givenBidRequest(ExtImpTradPlus.of(null, "zoneId"));
+
+        // when
+        final Result<List<HttpRequest<BidRequest>>> result = target.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getValue()).isEmpty();
+        assertThat(result.getErrors()).hasSize(1)
+                .containsOnly(BidderError.badInput("Invalid/Missing AccountID"));
+    }
```
Comment (AntoxaAntoxic) [inline src/test/java/org/prebid/server/bidder/tradplus/TradPlusBidderTest.java:83]: accoding to the json scheme null is impossible as an accountId value, so it make sense to check when accountId is blank " "

**Review by AntoxaAntoxic — CHANGES_REQUESTED** [review]

**Comment by SerhiiNahornyi** [comment]
@tradplus, could you please address the comments mentioned above?

**Comment by Compile-Ninja** [comment]
@tradplus There are conflicts in this PR. In addition to addressing the comments above, could you please merge the master branch as well?

**Comment by tradplus** [comment]
the issue has been fixed, please reopen this PR.

**Comment by tradplus** [comment]
A PR has been recreated：https://github.com/prebid/prebid-server-java/pull/3508.

**Pull request closed without merging**